src/nl/surf/eduhub_rio_mapper/rio/loader.clj

```diff
@@ -58,6 +66,9 @@
           (.getFirstChild)
           (.getTextContent)))
 
+(defn duo-string [x]
+  (str "duo:" (if (keyword? x) (name x) x)))
```

Comment:
Wordt volgens mij alleen gebruikt icm `duo-keyword` in `link` voeg ze dan maar samen.

en `(name x)` werkt ook voor strings, dus je kan hier `(str "duo:" (name x))` doen